### ISSUE TITLE
Hyperconverged setup should be simpler with arakoon as configuration manager

### ISSUE THREAD

**Issue by kinvaris** [openvstorage/framework#873]
Currently we get this on a setup: 

```
Please place a copy of the Arakoon's client configuration file at: /opt/asd-manager/config/arakoon_cacc.ini
Press enter to continue
```

When performing a setup it should be checked that this file exists `/opt/OpenvStorage/config/arakoon_cacc.ini` and perform the copy itself.

**Comment by khenderick** [comment]
Fixed in openvstorage/alba-asdmanager#108, packaged in openvstorage-sdm-1.6.3-rev.352.76af109, fix by @JeffreyDevloo

**Comment by JeffreyDevloo** [comment]
### Steps
- Got the packages on a fresh Ubuntu install using:

```
echo "deb http://apt.openvstorage.org unstable main" > /etc/apt/sources.list.d/ovsaptrepo.list
apt-get update

apt-get install volumedriver-no-dedup-server -y --force-yes
apt-get install openvstorage-hc -y --force-yes
apt-get install openvstorage-test openvstorage-health-check -y --force-yes
```
- Installed ovs via ovs setup
- Did not get the prompt

### Test result

Test passed.
